xml/System.IO.Enumeration/FileSystemName.xml

```diff
@@ -46,11 +46,11 @@
         <Parameter Name="ignoreCase" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="expression">To be added.</param>
-        <param name="name">To be added.</param>
-        <param name="ignoreCase">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="expression">The expression to match with.</param>
+        <param name="name">The name to check against the expression.</param>
+        <param name="ignoreCase"><see langword="true" /> to ignore case (default), <see langword="false" /> if the match should be case-sensitive.</param>
+        <summary>Verifies if the given expression matches the given name. Supports the following wildcards: '*' and '?'. The backslash character '\' escapes.</summary>
```

Comment:
The `\` character is not rendering on docs. I'll need to check with the eng team about this. Probably escaping would fix it for docs but could break IntelliSense.

Reply:
Could we use its html entity?
https://www.freeformatter.com/html-entities.html

`&#92;`

Reply:
We can try. Should we merge this and figure out on a separate PR how to fix this?

Reply:
Okay sure, let's get it merged. I can address it separately.